File: programmers/기초/간단한 식 계산하기.js
// /간단한 식 계산하기
function solution(binomial) {
  const cal = {
    "+": (a, b) => a + b,
    "-": (a, b) => a - b,
    "*": (a, b) => a * b,
  };
  const [a, op, b] = binomial.split(" ");
  return cal[op](Number(a), Number(b));
  // return cal[op](+a, +b);
}
// eval
const solution = eval;

File: programmers/python/간단한 식 계산하기.py
# 간단한 식 계산하기
def solution(binomial):
    a = binomial.split(" ")
    b = int(a[0])
    c = int(a[2])
    if a[1] == '+':
        return b + c
    elif a[1] == '-':
        return b - c
    else:
        return b * c